### DISCUSSION TITLE
Rspack Public Roadmap Discussion

### DISCUSSION THREAD

**Discussion by hardfist** [web-infra-dev/rspack#4422] [Announcements]
Rspack public roadmap is here https://github.com/orgs/web-infra-dev/projects/21/views/1
Tell us your feedback about our public roadmap
* what's your most wanted features
* what stops you from adopting Rspack in your project
* give some advices to Rspack

**Comment by h-a-n-a** [comment]
IMO, better diagnostics and error overlays should be prioritized.

**Reply by hardfist** [reply]
agree, we should list it separately in Q4

**Reply by bvanjoi** [reply]
I think the diagnostics should take two aspects into account: 1. the users who utilize rspack; 2. the developers who contribute to rspack or its ecosystem.

- For user, it should provide simple and clean diagnostics to directly pinpoint errors. And rust is indeed something we could learn from.
- And for developers, we should consider two aspects: 1. compatibility with the webpack ecosystem, and 2. developer experience.
